CMSgov/onemac
# remove waiver rais from the package list view

Story: https://qmacbis.atlassian.net/browse/OY2-20281
Endpoint: See github-actions bot comment

### Details

Filter Waiver RAI component types from the children list displayed on the package dashboard

### Changes

- Added filter to the getChildren method used to show subrows in the PortalTable for package list

### Test Plan

1. Login as state submitter
2. Navigate to package list on the waiver tab and locate a waiver with a known RAI Response
3. Verify the item either does not have an enabled expander or when expanded it does not show the RAI Response
